dockerfile_scripts/build_gdrcopy.sh

```diff
@@ -19,6 +19,6 @@ cd /tmp   && \
        ./packages/build-deb-packages.sh -t -k -d && \
     cd / && \
     dpkg -i \
-        /tmp/gdrcopy-$GDR_VER/libgdrapi_$GDR_VER-1_amd64.Ubuntu20_04.deb && \
```

Comment:
Is this necessary anymore? The base NGC images I used for the GH200 nodes already had libgdr* in /usr.

Reply:
I think we need this for our other image offerings. (i.e. Pytorch 2.0.1 and Pytorch 1.12) But this can be removed once we make these NGC images the default.